File: win-prog/ProgWinCS/Console Thyself/ConsoleAdder/ConsoleAdder.cs
//-------------------------------------------
// ConsoleAdder.cs © 2001 by Charles Petzold
//-------------------------------------------
using System;

class ConsoleAdder
{
     public static void Main()
     {
          int a = 1509;
          int b = 744;
          int c = a + b;

          Console.Write("The sum of ");
          Console.Write(a);
          Console.Write(" and ");
          Console.Write(b);
          Console.Write(" equals ");
          Console.WriteLine(c);

          Console.WriteLine("The sum of " + a + " and " + b + " equals " + c);

          Console.WriteLine("The sum of {0} and {1} equals {2}", a, b, c);
     }
}

File: win-prog/ProgWinVB/Console Thyself/ConsoleAdder/ConsoleAdder.vb
'---------------------------------------------
' ConsoleAdder.vb (c) 2002 by Charles Petzold
'---------------------------------------------
Imports System

Module ConsoleAdder
    Sub Main()
        Dim A As Integer = 1509
        Dim B As Integer = 744
        Dim C As Integer = A + B

        Console.Write("The sum of ")
        Console.Write(A)
        Console.Write(" and ")
        Console.Write(B)
        Console.Write(" equals ")
        Console.WriteLine(C)

        Console.WriteLine("The sum of " & A & " and " & B & " equals " & C)
        Console.WriteLine("The sum of {0} and {1} equals {2}", A, B, C)
    End Sub
End Module
